src/cljc/quil/core.cljc

```diff
@@ -2744,6 +2752,17 @@
       (let [pix-array (.toArray (.-pixels img))]
         (set! (.-stored-pix-array img) pix-array)
         pix-array))))
+#?(:cljs
+   (defn
+     ^{:requires-bindings true
+       :processing-name "plane()"
+       :category "Shape"
+       :subcategory "3D Primitives"
+       :added "1.0"}
```

Comment:
update version

Reply:
Done. Can you merge so I can push some more changes in a separate pull request?